src/Windows/Markdown/Editor/Impl/Preview/Parser/Extensions.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.Markdown.Editor.Preview.Parser {
     internal static class Extensions {
         public static string GetText(this LeafBlock block) {
-            if(block?.Lines == null) {
+            if(block?.Lines == null || block.Lines.Lines == null) {
```

Comment:
It means that `StringLineGroup` is a struct, so `block?.Lines.Lines == null` is enough